packages/flame/test/components/composed_component_test.dart

```diff
@@ -138,5 +138,21 @@ void main() {
       expect(child.rendered, true);
       expect(child.updated, true);
     });
+
+    test('initially same debugMode as parent', () {
+      final game = MyGame();
+      game.onResize(Vector2.all(100));
```

Comment:
just out of curiosity, do you need to call onResize here since your test doesn't care about it?

Reply:
It was just a copy of the previous test, this can be removed

Reply:
Ah, it can't be removed, because the game needs to have `hasLayout = true` before it can do `add`